src/Cache/Cache.php

```diff
@@ -58,8 +58,8 @@ public function set($file, $hash)
         if (!is_int($hash)) {
             throw new \InvalidArgumentException(sprintf(
                 'Value needs to be an integer, got "%s".',
-                is_object($hash) ? get_class($hash) : gettype($hash))
-            );
+                is_object($hash) ? get_class($hash) : gettype($hash)
+            ));
```

Comment:
I don't think so, this is the very problem this fixer is fixing. Not fixing it in that commit would make the commit invalid (the build for this commit wouldn't be green).